python_modules/libraries/dagster-dbt/dagster_dbt/dagster_dbt_translator.py

```diff
@@ -10,6 +10,13 @@
 
 
 class DagsterDbtTranslator:
+    """Holds a set of methods that derive Dagster asset definition metadata given a representation
+    of a dbt resource (models, tests, sources, etc).
+
+    This class is exposed so that methods can be overriden to customize how Dagster metadata is
+    derived from dbt metadata and configuration.
```

Comment:
Typo + more concise
```suggestion
    This class is exposed so that methods can be overridden to customize how Dagster asset definition metadata is derived.
```